docs/visual-basic/language-reference/statements/if-then-else-statement.md

```diff
@@ -49,7 +49,15 @@ End If
 ' Single-line syntax:  
 If condition Then [ statements ] [ Else [ elsestatements ] ]  
 ```  
-  
+
+## Quick links to example code
+
+This article includes several examples that illustrate uses of the `If`...`Then`...`Else`statement:
+
+* [Multiple-line syntax example](#multi-line)
+* [Nested syntax example](#nested)
+* [Single-line syntax example](#single-line)
+
 ## Parts  
  `condition`  
  Required. Expression. Must evaluate to `True` or `False`, or to a data type that is implicitly convertible to `Boolean`.  
```

Comment:
On line 65, there should be a space between Nullable and Boolean
if the expression is not True can be easily misread; much better to say "if the expression is \`False\`
no comma before "the \`Else\`"